src/main/java/frc/robot/poseestimator/limelight/VisionConstants.java

```diff
@@ -0,0 +1,78 @@
+package frc.robot.poseestimator.limelight;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+
+public class VisionConstants {
+
+    public static final String[] LIMELIGHT_NAMES = new String[]{
+            "limelight-front",
+            "limelight-back",
+            "limelight-gb"
+    };
+
+    public static final String LIMELIGHT_LOGPATH_PREFIX = "Limelight ";
+
+    public static final String MULTI_LIMELIGHT_LOGPATH = "MultiLimeLight/";
+
+    public static final String ESTIMATION_LOGPATH = "estimation ";
```

Comment:
```suggestion
    public static final String ESTIMATION_LOGPATH = "estimation";
```

Reply:
Added prefix to tha name of the const. As written, this is not the actual logpath